documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -41,9 +42,28 @@ export default {
 
 ## Options
 
+### config
+
+Path to your [Wrangler configuration file](https://developers.cloudflare.com/workers/wrangler/configuration/). If you would like to use a Wrangler configuration filename other than `wrangler.jsonc`, you can specify it using this option.
+
+### platformProxy
+
+Preferences for the emulated `platform.env` local bindings. See the [getPlatformProxy](https://developers.cloudflare.com/workers/wrangler/api/#parameters-1) Wrangler API documentation for a full list of options.
+
+### fallback
```

Comment:
I just wanted to mention, in case you are unaware that workers static assets is also going to support this sort of functionality 🙂 (https://github.com/cloudflare/workers-sdk/pull/8501) it's not publicly announced nor documented yet though